app/main/helpers/frameworks.py

```diff
@@ -65,7 +64,7 @@ def get_required_fields(all_fields, answers):
         required_fields.add('SQ3-1k')
 
     # If you answered No to question 26 (established in the UK)
-    if not answers.get('SQ5-2a'):
+    if answers.get('SQ5-2a') is False:
```

Comment:
Identity checks tend to behave differently in different interpreters (e.g. PyPy) based on how frames/object interning works. `is None` is ok since `None` is strictly defined as a singleton, but everything else is a bit fuzzy. I think `is False` could be fine, but I'd still rather avoid this when possible.